app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, campaign: String)
+
+
+object DigitalEdition {
+
+  trait UrlConfig {
+    val externalSubscriptionUrl: String
+  }
+
+  object UK extends DigitalEdition("uk", "UK", "£11.99", "dis_2408")
+  object US extends DigitalEdition("us", "US", "$19.99", "dis_2378")
+  object AU extends DigitalEdition("au", "Australia", "$21.50", "dis_2379")
+  implicit lazy val defaultUrlConfig: UrlConfig = Config
+
+  def getRedirect(edition: DigitalEdition)(implicit urlConfig: UrlConfig): String = {
```

Comment:
I think you are reaching out to heavy artillery (i.e. `implicit urlConfig: UrlConfig`) to do something fairly simple here. Why not just provide a base url as the first function argument?

```scala
def getRedirect(baseUrl: URL, edition: DigitalEdition): URL = ???
```
Also, Perhaps it is a good idea to use `java.net.URL` instead of String, at least for the baseUrl value (Typesafe config provides a convenient [parseUrl](https://typesafehub.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#parseURL-java.net.URL-) method to read these from config files?